# Horizon Points
# This Message is published by Horizon Plotter

# left horizon point in image 2D
geometry_msgs/Vector2 pt_left

# right horizon point in image 2D
geometry_msgs/Vector2 pt_right

# center horizon point in image 2D
geometry_msgs/Vector2 pt_center

# center max depth point in image 2D
geometry_msgs/Vector2 pt_max_depth
